### ISSUE TITLE
8.0.1 release planning

### ISSUE THREAD

**Issue by bluetech** [pytest-dev/pytest#11969]
I plan to release pytest 8.0.1 this weekend (Friday or Saturday), we have fixed many regressions.

Current nice-to-haves that are not yet merged:

- #11917 (waiting on author)
- #11959 (maybe; might be only relevant for 8.1, need to check)
- #11920 (waiting on review)
- #11936 (maybe; might wait for next patch release depending on review time and whether @nicoddemus would want to backport)

**Comment by 0x78f1935** [comment]
Hi @bluetech, this probably doesn't have anything to do with Pytest, but with pytest-coverage. Yet I was wondering if this version might solve my [issue](https://github.com/pytest-dev/pytest-cov/issues/628)

**Comment by bluetech** [comment]
@0x78f1935 The issue you're describing doesn't sound familiar to me, so probably not. You say in the issue that it happens on py3.12 but not on py3.11 so it sounds like maybe a Python issue? (I know that py3.12 had some coverage changes so maybe try running `python manage.py test` with coverage and see if it reproduces)

**Comment by 0x78f1935** [comment]
@bluetech Thank you for your response! I don't think the coverage plugin was compatible with the Django management file, but I'll have to check that again to verify this statement. If I have any updates, I'll update the issue!
